workflows/pr-build-php73.yml

```diff
@@ -0,0 +1,48 @@
+name: PHP 7.3 tests
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Setup PHP with specific version of PECL extension
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.3'       
```

Comment:
This should be done as a matrix so we only have one workflow to manage.

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs